app/views/staff/session_formats/create.js.erb

```diff
@@ -11,4 +11,5 @@
   $('#config-modal .errors').append('<%=  session_format.errors.full_messages.join(", ") %>');
```

Comment:
Grrr, I have been fighting with this on and off this entire card. I really thought it was finally fixed. These modals are so buggy.